public/styles.css

```diff
@@ -0,0 +1,39 @@
+html, body {
+	background-color: #eee;
+}
+
+.content {
+	background-color: #fff;
+	padding: 20px;
+	margin: 0 -20px; /* negative indent the amount of the padding to maintain the grid system */
+	border-radius: 0 0 6px 6px;
+	box-shadow: 0 1px 2px rgba(0, 0, 0, .15);
+}
+
+.page-header {
+	background-color: #f5f5f5;
+	padding: 20px 20px 10px;
+	margin: -20px -20px 20px;
+}
+
+footer ul {
+	list-style-type: none;
+}
+
+.error {
+	display: flex;
+}
+
+.error > .icon {
+	flex: 1;
+	font-size: 2rem;
+}
+
+.error > .message {
+	flex: 20;
+	padding-top: 0.4rem;
+}
+
+.error p {
+	margin: 0;
+}
```

Comment:
Nitpick, maybe also add
```
.error glyphicon {
    top: 9px;
}
```
to make the (X) icon in the error message more vertically aligned with the text. Tested in Chrome and Firefox.

Reply:
Good point. Actually, if we `align-items: center;` then it's always in the right place (I think).